src/IB/IBRedundantInitializer.cpp

```diff
@@ -1496,8 +1496,11 @@ IBRedundantInitializer::getVertexBdryMassSpec(const std::pair<int, int>& point_i
 const std::vector<double>&
 IBRedundantInitializer::getVertexDirectors(const std::pair<int, int>& point_index, const int level_number) const
 {
-    if (!d_init_director_and_rod_on_level_fcn)
+    if (static_cast<size_t>(point_index.second) >= d_directors[level_number][point_index.first].size())
     {
+        TBOX_WARNING("Could not find enough director vectors on level " << level_number << " for structure "
```

Comment:
Instead of emitting a warning, how about just emitting an error?

It seems like we should require that the correct number of director vectors is always provided, or perhaps allow for a special case in which _no_ directors are provided --- and in that case alone default to the standard basis vectors.

Reply:
We do a lot of error checking when we read the input files (or run the function) to make sure everything is consistent sizes, e.g. `num_vertex == num_directors`. This warning will only be emitted if no directors are registered (either through input file or function), and then it will be emitted as many times as there are vertices. The idea was to make the log file clear that something is almost definitely wrong.

I think emitting an error is the best thing to do. The only situation in which initializing directors to the standard basis is valid is if the structure's tangential vector is in the z-direction.

Reply:
I guess the other option would be to initialize the directors so that the structure is in equilibrium...?